cmd/main.go

```diff
@@ -45,14 +42,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	digest := *digestStr
 	if _, err := hex.DecodeString(digest); err != nil && len(digest) != 64 {
```

Comment:
I think this should be `err != nil || len() != 64`. Maybe seperate the 2 checks to give different error message.
How about defining 64 as a variable? (it's pretty obvious tbh already :-))

Reply:
we could also use go-validator libs to validate sha256, but for now i can just use sha256 suze